inc/classes/class-srm-post-type.php

```diff
@@ -49,6 +49,8 @@ public function setup() {
 		add_filter( 'default_hidden_columns', array( $this, 'filter_hidden_columns' ), 10, 1 );
 		add_action( 'admin_enqueue_scripts', array( $this, 'load_resources' ), 10, 0 );
 		add_action( 'wp_ajax_srm_validate_from_url', array( $this, 'srm_validate_from_url' ), 10, 0 );
+		add_action( 'wp_ajax_nopriv_srm_autocomplete', array( $this, 'srm_autocomplete' ), 10, 0 );
```

Comment:
@bmarshall511 Is there any reason we are handling Ajax requests from unauthenticated users?